Doc/requirements.txt

```diff
@@ -1,12 +0,0 @@
-# Requirements to build the Python documentation
```

Comment:
`Doc/requirements.txt` is still needed:

https://github.com/python/cpython/runs/4649987492?check_suite_focus=true#step:7:30

Reply:
@hugovk thank you! 👍